README.md

```diff
@@ -26,6 +26,7 @@ ______________________________________________________________________
 <details open>
 <summary><b>2024</b></summary>
 
+- \[2024/06\] Support VLMs in PyTorch engine, such as CogVLM2, Mini-InternVL, LlaVA-Next
```

Comment:
The support for DeepSeek V2 in the PyTorch Engine is a major feature and highlight. Perhaps we may also mention it in the news.

Reply:
Oh, my bad